packages/glimmer-runtime/lib/dom/change-lists.ts

```diff
@@ -126,6 +134,27 @@ export const InputValuePropertyChangeList = {
   }
 };
 
+export const TextAreaValueChangeList = {
+  setAttribute(dom: DOMHelper, element: Element, attr: string, value: Opaque) {
+    let textArea = <HTMLTextAreaElement>element;
+    let normalizedValue = normalizeTextValue(value);
+
+    if(textArea.childNodes.length === 0) {
+      let textNode = dom.createTextNode(normalizedValue);
+      textArea.appendChild(textNode);
+    } else {
+      let currentValue = textArea.value;
```

Comment:
Well, technically `<textarea>` doesn't have an attribute for `value`.  So actually having `<textarea value={{someThing}}></textarea>` seems wrong?

I was under the impression that writing a manual text area would look like:

```js
<textarea>{{someValue}}</textarea>
```

Though I guess this also has issues, because now updates won't work properly?